### ISSUE TITLE
Request for "logout"

### ISSUE THREAD

**Issue by Kynareth62** [Contex/XenAPI#32]
Request : Add "logout" (important for external website).
Thank you :)

**Comment by tempfirstuser** [comment]
Is there a way to log out a user somehow?

**Comment by NaakkaDev** [comment]
Its possible to "log out" the user by deleting the cookie `xf_session`. In some cases at least.

**Comment by tempfirstuser** [comment]
Temeez, I can't do that. Of course, it was my first step to delete cookie (even cookies) xf_session, but it has no effect.

**Comment by NaakkaDev** [comment]
Yeah, it is a bit random to be honest. Hopefully 1.4.3 comes out soon ;)
